src/components/ParcelsList.vue

```diff
@@ -187,7 +189,8 @@ export default {
   computed: {
     ilots() {
       // first reduce to group parcels by ilots
-      let reduced = this.parcels.features.reduce(
+      let reduced = reduce(
```

Comment:
J'ai remis comme avant, car le `reduce` de lodash sait itérer sur des objets également.